qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -31,8 +31,7 @@
         - @unplug_port:
             serials += " vc1 vc2"
             serial_type_vc1 = virtserialport
-            serial_type_vc2 = virtconsole
-            file_transfer_serial_port = vc1
+            serial_type_vc2 = virtserialport
```

Comment:
Not a big issue,but as the common usage for this feature,  `vs1  vs2` is better ?

Reply:
Have you reached an agreement with whql team, use two `virtserialport`, remove `virtconsole` ?

Reply:
> Have you reached an agreement with whql team, use two `virtserialport`, remove `virtconsole` ?

Yes, have confirmed with xiagao.

Reply:
Ok, please provide the test result with both linux and windows guest.

Reply:
> Not a big issue,but as the common usage for this feature, `vs1 vs2` is better ?

update and thanks.